examples/cryptorealtime/terraform-setup/startup.tpl

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Ivo Galic
```

Comment:
If this is being submitted to the GoogleCloudPlatform repository in CLA, I believe this should be copyright Google.

@ryanmcdowell  is that right?